examples/datadog-express-apm/container-agent/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "datadog-express-apm",
+  "name": "container-agent",
```

Comment:
I believe it's better if you add something related to Datadog like "datadog-container-apm"